dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -133,6 +134,9 @@ public String getFinalTruncatedSqlExpression(String tableLoc) {
     if (this.getMultiFieldNames() != null) {
       return "left(" + getFinalSqlExpression(tableLoc) + ",600)";
     }
+    if (!this.isStringType()) {
+      return this.fieldPath;
+    }
```

Comment:
If the javadoc says "stringType=false does not truncate" then it should go before the `if (this.getMultiFieldNames() != null)` that also truncates.

Reply:
Would it ever possible to combine the two?